en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup-with-tm-separated.md

```diff
@@ -626,62 +693,38 @@ Configure the Traffic Manager to communicate with the Control Plane.
     **Connecting the Traffic Manager to the Control Plane node**:
 
     ```toml tab="Control Plane with High Availability"
- 
     # Event Hub configurations
     [apim.event_hub]
     enable = true
     username = "$ref{super_admin.username}"
     password = "$ref{super_admin.password}"
     service_url = "https://[control-plane-LB-host]/services/"
     event_listening_endpoints = ["tcp://control-plane-1-host:5672", "tcp://control-plane-2-host:5672"]
-
-    [[apim.event_hub.publish.url_group]]
-    urls = ["tcp://control-plane-1-host:9611"]
-    auth_urls = ["ssl://control-plane-1-host:9711"]
-
-    [[apim.event_hub.publish.url_group]]
-    urls = ["tcp://control-plane-2-host:9611"]
-    auth_urls = ["ssl://control-plane-2-host:9711"] 
-
     ```
  
     ```toml tab="Single Control Plane"
- 
     # Event Hub configurations
     [apim.event_hub]
     enable = true
     username = "$ref{super_admin.username}"
     password = "$ref{super_admin.password}"
     service_url = "https://[control-plane-host]/services/"
     event_listening_endpoints = ["tcp://control-plane-host:5672"]
-
-    [[apim.event_hub.publish.url_group]]
-    urls = ["tcp://control-plane-host:9611"]
-    auth_urls = ["ssl://control-plane-host:9711"]
-
     ```
 
+    !!! Info
+        With `event_listening_endpoints`, traffic manager is subscribed to the JMS stream of both event hubs. Once a policy related event is received, it will pull the execution plans from the `service_url`.
```

Comment:
```suggestion
        With `event_listening_endpoints`, the Traffic Manager is subscribed to the JMS stream of both event hubs. Once a policy-related event is received, it will pull the execution plans from the `service_url`.
```